src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Figure implements FigureDraw {

```

Comment:
```suggestion
public class Figure implements FigureDraw, FigureArea {
```
тоді аналогічно не потрібно буде в наслдниказ прописувати цей інтерфейс в сигнатурі класу